### PR TITLE
issue #78  use logo in header and footer

### PR DESCRIPTION
**Pull request by letsmailvjkumar**
issue #78
added the log image location which was different.

### CODE REVIEW

**Comment by adityaraute** [comment]
The issue is not linked to the PR by only mentioning it. You have to either use the term 'Fixes #' or link it on the right hand sidebar.
This is why we have PR templates in place.

File: index.html
Comment (adityaraute) [inline index.html:1]: You are using the larger picture. You have to use the `favicon` image from the brand directory.

Additionally, you are only changing the logo on `index.html`. You should also change the logo on other webpages.

**Review by adityaraute — CHANGES_REQUESTED** [review]
There are a few requested changes, please correct them; only then can the PR be merged.
Let us know if you need any help

**Comment by letsmailvjkumar** [comment]
Thanks for the detailed report. I will fixed it.

**Pull request closed without merging**